doc/registration.md

```diff
@@ -0,0 +1,96 @@
+# Data registration queries
+
+## By Registration ID
+
+### Overview
+
+These queries enable clients to find a single Registration by its unique ID.  
+
+### Syntax 
+        protocol://ws-entry-point/registration/id/{registrationId}
+
+### Example
+
+Retrieve all Registrations for the Registration with unique ID 'abcd-efgh-ijkl'
```

Comment:
Should this read "Retrieve the Registration with unique ID 'abcd-efgh-ijkl'"? 

If not, then what is a Registration of Registrations?

Reply:
thanks, resolved